public/css/common.less

```diff
@@ -316,87 +297,9 @@ div.show-more {
   }
 }
 
-.item-table {
-  &.table-layout {
-    &.hostgroup-table {
-      --columns: 2;
-    }
-
-    &.servicegroup-table {
-      --columns: 1;
-    }
-
-    &.user-table, // TODO: make them lists.....
-    &.usergroup-table {
-      --columns: 0;
-    }
-  }
-}
-
-.hostgroup-table,
-.servicegroup-table,
-.usergroup-table,
-.user-table {
-  .title .column-content > * {
-    display: block;
-    max-width: fit-content;
-  }
-
-  .object-statistics-total {
-    width: 3.75em;
-  }
-}
-
-.controls .hostgroup-table-row,
-.controls .servicegroup-table-row,
-.controls .usergroup-table-row,
-.controls .user-table-row {
-  .title .column-content {
-    display: inline-flex;
-    align-items: center;
-
-    > :first-child {
-      flex: 0 1 auto;
-    }
-
-    > :last-child {
-      flex: 1 1 auto;
-    }
-
-    .subject {
-      margin-right: .5em;
-    }
-  }
-
-  .vertical-key-value {
-    br {
-      display: none;
-    }
-
-    .key {
-      padding-left: .417em;
-    }
-
-    .value {
-      vertical-align: middle;
-    }
-  }
-}
-
-.history-list,
-.objectHeader {
+.item-layout.history {
   .visual.small-state-change .state-change {
-    padding-top: .25em;
-  }
-}
-
-.comment-popup {
-  .comment-list .main {
-    // This is necessary to limit the visible comment lines
-    // because the popup is shown in detailed list mode only
-    .caption {
-      height: 3em;
-    }
+    padding-top: .25em; // TODO: why, where is this used?
```

Comment:
Here:
![image](https://github.com/user-attachments/assets/312d82a4-6ceb-490d-a0b8-59bd53930648)

Reply:
Currently mis-placed, since this is a host header and the current rule doesn't apply.